core/Plugin/Controller.php

```diff
@@ -809,6 +819,32 @@ protected function addCustomLogoInfo($view)
         $view->customFavicon = $customLogo->getPathUserFavicon();
     }
 
+    /**
+     * Set the template variables to show the what's new popup if appropriate
+     *
+     * @param View $view
+     * @return void
+     */
+    protected function showWhatIsNew(View $view): void
+    {
+        $view->whatisnewShow = false;
+
+        if (isset($view->hideWhatIsNew) && $view->hideWhatIsNew) {
+            return;
+        }
+
+        $model = new UsersModel();
+        $user = $model->getUser(Piwik::getCurrentUserLogin());
+        if ($user) {
```

Comment:
Small thing, if you do `if (!$user)` then you can return early and have less nesting

Reply:
Good point, that'd be a bit easier to read :+1: